packages/server/src/build.rs

```diff
@@ -550,6 +553,33 @@ impl Server {
 		}
 	}
 
+	async fn try_get_build_log_blob(&self, id: &tg::build::Id) -> Result<Option<tg::blob::Id>> {
```

Comment:
I don't have a better name for this function, since it's not really getting the `log`, it's getting the `log` field in the `json` field of the build in the local database.